src/components/mnemonic/mnemonic-create/mnemonic-create.tsx

```diff
@@ -43,13 +44,22 @@ export const MnemonicCreate: FC<MnemonicProps> = ({ value, isError, onChangeText
         style={StyledTextInputStyles.mnemonicInput}
         onBlur={onBlur}
         onChangeText={onChangeText}
+        testID={testID}
       />
       <View style={MnemonicStyles.buttonsContainer}>
-        <ButtonSmallSecondary title="GEN NEW" onPress={handleGenerateNewButtonPress} />
+        <ButtonSmallSecondary
+          title="GEN NEW"
+          onPress={handleGenerateNewButtonPress}
+          testID={CreateNewWalletSelectors.GenNewSeedButton}
+        />
         {isString(value) ? (
           <>
             <Divider size={formatSize(8)} />
-            <ButtonSmallSecondary title="COPY" onPress={() => copyStringToClipboard(value)} />
+            <ButtonSmallSecondary
+              title="COPY"
+              onPress={() => copyStringToClipboard(value)}
+              testID={CreateNewWalletSelectors.CopyButton}
```

Comment:
You are using `CreateNewWalletSelectors` inside `MnemonicCreate` component. 
This's wrong.  Selectors should only be used in component where they are declared